source/linux/Foundational_Components/Power_Management/pm_rtc_ddr.rst

```diff
@@ -0,0 +1,94 @@
+#########################
+RTC-Only and RTC+DDR Mode
+#########################
+
+AM437x supports two RTC modes, RTC+DDR and RTC-Only mode. RTC+DDR Mode is similar to the
+Suspend/Resume above but only supports wake by the Power Button present
+on the board or from an RTC ALARM2 Event. RTC-Only mode supports the
+same wake sources, however DDR context is not maintained so a wake event
+causes a cold boot.
+
+RTC-Only mode is supported on:
+
+-  AM437x GP EVM
+-  AM437x SK EVM
+
+RTC+DDR mode is supported on:
+
+-  AM437x GP EVM
+
+.. rubric:: RTC+DDR Mode
+
+The first step in using RTC+DDR mode is to enable off mode by typing the
+following at the command line:
+
+.. code-block:: console
+
+        $ echo 1 > /sys/kernel/debug/pm_debug/enable_off_mode
```

Comment:
https://github.com/TexasInstruments/processor-sdk-doc/blob/master/CONTRIBUTING.md#indentation-and-whitespace